config/database.yml.example

```diff
@@ -1,19 +0,0 @@
-default: &default
```

Comment:
qrde ten plik usunąłem chyba jakoś zaraz po naszym pierwszym spotkaniu, wydaje mi się, że coś mi ten plik mieszał, ale to równie dobrze mogło mi się tylko wydawać...